SpatialGDK/Source/SpatialGDK/Public/SpatialView/ViewDelta.h

```diff
@@ -25,7 +31,10 @@ class ViewDelta
 	void Clear();
 
 private:
+	// todo wrap world command responses in their own record?
```

Comment:
Again, not a fan of scattered todo comments, would prefer tickets otherwise they're unlikely to be actioned

Reply:
For a lot of these they are meant to be notes to the reader of what to expect in the coming PRs as this is only partially finished. So I'm not sure tickets are the right thing here. We can make them not todos though. We can also remove speculative ones if you'd rather.

Reply:
We should explicitly have a pass over this when the last op is included to convert all todos to tickets, but until that stage I don't mind the todos.